### PR TITLE
core(preload): temporarily disable all preload advice

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Temporarily moves all our preload audits to always return `notApplicable` while the preload fix is being evaluated. See #11960 for long-term plan and more discussion.

**Related Issues/PRs**
ref https://github.com/GoogleChrome/lighthouse/issues/11960

FYI @khempenius @addyosmani @spanicker

### CODE REVIEW

**Comment by connorjclark** [comment]
can you summarize AIs needed for v9 in the tracker? https://github.com/GoogleChrome/lighthouse/issues/12614

**Comment by patrickhulce** [comment]
> can you summarize AIs needed for v9 in the tracker? #12614

Done 👍

**Comment by patrickhulce** [comment]
friendly ping @adamraine :)

File: lighthouse-core/test/results/sample_v2.json [hunk 1]
```diff
@@ -6414,12 +6381,6 @@
         "duration": 100,
         "entryType": "measure"
       },
-      {
-        "startTime": 0,
-        "name": "lh:computed:LanternLargestContentfulPaint",
-        "duration": 100,
-        "entryType": "measure"
-      },
```
Comment (adamraine) [inline lighthouse-core/test/results/sample_v2.json:6422]: Were we making an extra call to this that we didn't need to?

**Review by adamraine — APPROVED** [review]
Overall LGTM

File: lighthouse-core/test/results/sample_v2.json (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/test/results/sample_v2.json:6422]: No, it was necessary. The preload LCP image audit makes an explicit *lantern* LCP request in order to get the LCP graph but our sample results here uses observed metrics for LCP itself which never requests the lantern computed artifact.
